src/loader.c

```diff
@@ -584,7 +612,8 @@ bool load_auth_file(const char *fn)
 		*p++ = 0; /* tag password end */
 
 		/* send them away */
-		unquote_add_user(user, password);
+		user = unquote_add_user(username, password);
+		user->from_auth_file = true;
```

Comment:
Flag that user was from `auth_file` to know whether to send off `auth_query` upon login.